atm.js

```diff
@@ -1,21 +1,50 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+$(document).ready(function(){
+
+var checkingBalance = $("#checking_balance");
+var checkingEntry = $("#checkingEntry");
+var savingsBalance = $("#savings_balance");
+var savingsEntry = $("#savingsEntry");
+
+  $('#depositChecking').on("click", function() {
+    checkingBalance.text("$" + (parseInt(checkingBalance.html().replace("$", "")) + parseInt(checkingEntry.val())));
+    checkingEntry.val('');
+    checkingEntry.focus();
+  });
+
+  $('#withdrawChecking').on("click", function() {
+    checkingBalance.text("$" + (parseInt(checkingBalance.html().replace("$", "")) - parseInt(checkingEntry.val())));
+    checkingEntry.val('');
+    checkingEntry.focus();
+    if (checkingEntry.val() > parseInt(checkingBalance.html().replace("$", ""))) {
```

Comment:
What are we checking for here?  Please clarify with a variable for function that returns the value you want.